### PR TITLE
Display smoke tests results

### PR DESCRIPTION
**Pull request by tisnik**
Now the dashboard displays results of smoke tests run against production and stage as well:
https://fabric8-analytics.github.io/dashboard/dashboard.html

It is the last item for https://github.com/openshiftio/openshift.io/issues/1099

### CODE REVIEW

File: dashboard/src/smoke_tests.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        """Read results generated by all smoke tests."""
+        for system, filenames in SmokeTests.INPUT_FILES.items():
+            input_file = filenames["results"]
+            with open(input_file) as fin:
+                self._results[system] = fin.read().strip() == "0"
+
+    @property
+    def results(self):
+        """Getter for the 'results' attribute."""
+        return self._results
```
Comment (jpopelka) [inline dashboard/src/smoke_tests.py]: One improvement that just came to my mind is that you could also read the results in this *getter* if they haven't been read before, i.e.

```python
   @property
   def results(self):
       if not self._results:
           self._results = self.read_results()
       return self._results
```

so you don't have to call the `read_results()` explicitly before accessing `results` property.

**Comment by tisnik** [comment]
[test this]

File: dashboard/src/smoke_tests.py (hunk 1 above)
Comment (tisnik) [reply dashboard/src/smoke_tests.py]: yep, let's do this. TY!

**Comment by jpopelka** [comment]
Can you check the tests ?

**Pull request merged**